src/Identities.ts

```diff
@@ -36,9 +36,9 @@ export class Identities {
    * Register an Identity
    *
    * @note must be a CDD provider
-   * @note this may create [[AuthorizationRequest | Authorization Requests]] which have to be accepted by
-   *   the corresponding [[Account | Accounts]] and/or [[Identity | Identities]]. An Account or Identity can
-   *   fetch its pending Authorization Requests by calling `authorizations.getReceived`
+   * @note this may create {@link AuthorizationRequest Authorization Requests} which have to be accepted by the `targetAccount`.
```

Comment:
```suggestion
   * @note this may create {@link AuthorizationRequest | Authorization Requests} which have to be accepted by the `targetAccount`.
```